README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/hiqua/securify.svg?branch=master)](https://travis-ci.com/hiqua/securify)
```

Comment:
Yes I need to change this, but as of now it won't work anyway because travis does not have access to the eth-sri organization. This is actually a blocker for this travis CI thing.